pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -191,7 +213,34 @@ func (c *InstallerController) getStaticPodState(ctx context.Context, nodeName st
 	if err != nil {
 		return staticPodStatePending, "", "", nil, time.Time{}, err
 	}
+
 	revision := pod.Labels[revisionLabel]
+
+	// one state for the fallback case. We don't distinguish between fallback failing, running or ready.
+	if fallbackForRevision, ok := pod.Annotations[annotations.FallbackForRevision]; ok {
+		reason := "Unknown"
+		message := "unknown"
+		if s, ok := pod.Annotations[annotations.FallbackReason]; ok {
+			reason = s
+		}
+		if s, ok := pod.Annotations[annotations.FallbackMessage]; ok {
+			message = s
+		}
+
+		// Return fallbackForRevision intentionally. The revision of the fallback pod does not matter. We are technically
+		// in the rollout of fallbackForRevision, not of revision (the fallback operand pod).
+		//
+		// Return creationTimestamp intentionally. This timestamp must be constant. It is written to the static operator nodeStatus and
+		// every time it changes, the fail counter will increase. We define fallback pod creation date as the fallback time. Not 100%
+		// precise, but good enough and best we can get.
+		return staticPodStateFallback,
```

Comment:
do we want to say something about retrying a failed revision in the "main" loop? similar to 

```
if installerPodFailed {
				klog.Infof("Will retry %q for revision %d for the %s time because %s", currNodeState.NodeName, currNodeState.TargetRevision, nthTimeOr1st(newCurrNodeState.LastFailedCount), reason)
			}
```

Reply:
done